src/components/atoms/ConversationMessage.vue

```diff
@@ -9,8 +9,8 @@
     @keydown.prevent.exact.tab="nextElement($event)"
     aria-live="polite"
   >
-    <!-- <span class="sr-only">{{ (isUser ? "You " : senderName) }} said:</span> -->
-    <h4
+    <span class="sr-only"> </span>
+    <span
```

Comment:
axe objects to this with a comment of "ARIA attribute is not well supported on the element and the text content will be used instead: aria-label"

Reply:
there is a better explanation here. https://dequeuniversity.com/rules/axe/4.3/aria-allowed-attr?application=AxeChrome

Reply:
is there any reason that this can't be an h3 for the chat message? It seemed like the headings were a no go for the item in the inbox but it wasn't clear to me if it was possible in the chat.

Reply:
The chat messages aren't semantically headings, so keeping them as headings at all wouldn't really work.

Reply:
I resolved it by adding an sr-only span and also added aria-hidden so that the content isn't repeated